cmd/river/riverbench/river_bench.go

```diff
@@ -94,17 +96,17 @@ func (b *Benchmarker[TTx]) Run(ctx context.Context) error {
 		// values against the wall, they perform quite well. Much better than
 		// the client's default values at any rate.
 		FetchCooldown:     2 * time.Millisecond,
-		FetchPollInterval: 5 * time.Millisecond,
+		FetchPollInterval: 20 * time.Millisecond,
```

Comment:
Yeah ... I still wouldn't commit to any particular numbers because I can see fairly wild swings in performance between runs, but I played with these numbers more and I think these ones gave me slightly better results.